packages/shared/src/lib/constants.ts

```diff
@@ -1,9 +1,8 @@
 declare const navigator: Navigator & { brave?: { isBrave: unknown } };
-export const faq = 'https://github.com/dailydotdev/daily/blob/master/FAQs.md';
+export const faq = 'https://docs.daily.dev/faqs';
 export const requestFeature =
-  'https://github.com/dailydotdev/daily/issues/new?assignees=&labels=Type%3A+Feature&template=---feature-request.md&title=%F0%9F%A7%A9+FEATURE+REQUEST%3A+';
-export const reportIssue =
-  'https://github.com/dailydotdev/daily/issues/new?assignees=&labels=Type%3A+Bug&template=---bug-report.md&title=%F0%9F%90%9B+BUG%3A+';
+  'https://github.com/dailydotdev/daily/discussions/new?category=feature-request';
+export const reportIssue = 'https://it057218.typeform.com/to/zN8B5Vog';
```

Comment:
have a cname record or redirect for this? Something like https://report-issue.daily.dev this way we can update the form in the future without changing the product.

Reply:
Good point, we actually use an external system to manage links.
I'll raise this with product 👍